### ISSUE TITLE
lad 4 - Part 3

### ISSUE THREAD

**Issue by ecking** [DS4PS/cpp-523-sum-2020#12]
Hello, 

I was hoping I could get a little bit more explanation on how to answer part 3 in question 2 on the lab. 

It says explain the direction of bias. But I see the sign of bias is present so I'm a bit confused on what the expectation is for an answer here? 

I'm planning on joining the call tomorrow at 2, so if we can touch on this question's meaning, that would be appreciated.

**Comment by lecy** [comment]
This slide presents some info on the question: 

![image](https://user-images.githubusercontent.com/1209099/84493182-35ee4d80-ac5c-11ea-9e9f-0a85d3d68ba2.png)

Since bias is (a1)(B2) then it can be any of the following:
```
(+)(+) = +
(+)(-) = -
(-)(+) = +
(-)(-) = -
```
The problem is you won't have the ability to calculate a1 and B2 direct since that would require the omitted variable. But can you determine the sign of both without having all of the data? 

It turns out we are pretty good at intuiting the sign of a correlation. We are just bad at predicting the strength. 

If you can put a sign on both, then you will at the very least know whether b1 is too big or too small.

**Comment by ecking** [comment]
Am I oversimplifying this? Isn’t the sign of bias the direction of bias? I.e left or right

**Comment by lecy** [comment]
Yes, once you know the sign you know whether the naive coefficient will over-estimate or under estimate the true impact (depending on whether B1 is to the left or right of b1). 

It's just a way to bound uncertainty when you know you are missing an important variable in your study, but you can sign the correlations. If you can do so then when presenting results you can say something like, "we know our estimate is biased because we fail to include factor X in the model, but if we were able to add the variable it would actually increase the effect because we know that the signs of the correlations would be X and X, and therefore the bias is negative and the unbiased coefficient would be larger than our current estimate".
